src/PrestaShopBundle/Resources/views/Admin/Improve/Design/positions.html.twig

```diff
@@ -103,7 +113,7 @@
               </header>
 
               {% if hook['modules_count'] %}
-                <section class="module-list">
+                <section class="module-list" {% if not hook['active'] %} style="opacity:50%" {% endif %}>
```

Comment:
Hardcoded `style` 😋  ?

I'm bad at CSS so maybe I can approve this one but you won't be able to pass through @PierreRambaud and @Progi1984 :trollface:

Reply:
When I did I was scared that @NeOMakinG  would hit me ... 😅
Do you know of a bootstrap class for handling "opacity"?  Something like: `class = "opacity-0.5"` ? Sorry, I'm bad at CSS 😅

Reply:
in `admin-dev/themes/new-theme/scss/pages/_modules.scss` there is a `.module-list` you can add a class inside to handle this behavior. We shouldn't use bootstrap helpers (regarding a previous discussion with @NeOMakinG )

Reply:
I think the best solution would be a `disabled` or `inactive `class (`.module-list.disabled`) that has the styles, today it would be `opacity: 50%`, but in the future it could be different.

Reply:
I'm late for this PR that is already merged.